_preload.lua

```diff
@@ -34,6 +34,10 @@ newaction {
 		}
 	},
 
+	toolset = iif(os.target() == "windows", "msc-v142", -- Visual Studio 2019
+						iif(os.target() == "macosx", "clang",
+																				"gcc")),
+
```

Comment:
indentation looks bananas here, and we're also calling os.target() twice for non-windows systems. consider storing the result of os.target() into a local variable, or even `qmake._TARGET`

Reply:
tab with 2 space is ok... :/

Reply:
Local variables would not leak into the global namespace as long as you mark them as `local`.

And you can't know for sure that `os.target()` will stay a simple getter. This is more about using good programming practices. I'll let that one slide, though so longas you fix the alignment. Tabs are used for indentation, but you can use spaces for alignment if you wish. Another way to solve it is make a toolset look-up table and pass in the result of `os.target()`.

Reply:
Changed.